content/applications/websites/ecommerce/checkout_payment_shipping/ewallet_giftcard.rst

```diff
@@ -0,0 +1,122 @@
+======================
+eWallet and gift cards
+======================
+
+You can enable **eWallets** and **gift cards** for your customers on your eCommerce, which they can
+use to for their online and in-store shopping. To allow eWallets and gift cards for your eCommerce,
+head to :menuselection:`Website app --> eCommerce` and in the :guilabel:`Shop - Products` category,
+enable :guilabel:`Discounts, Loyalty & Gift Card`.
+
+eWallet
+=======
+
+eWallet allows customers to save their credits on their online account and use these credits as a
+payment method when buying items on your online store. As multiple redeems for the same order are
+not possible in Odoo, eWallets can also be used by customers to centralize multiple :ref:`gift cards
+<ewallet_gift/gift-cards>` and accumulate the credits.
+
+Before creating an eWallet program, it is necessary to first create an eWallet **top-up** product.
+To do so, go to :menuselection:`Website app --> eCommerce --> Products` and :guilabel:`Create` a new
+product. On the product template, configure the product as such:
+
+- :guilabel:`Can be sold`: enabled;
+- :guilabel:`Product Type`: set as :guilabel:`Service`;
+- :guilabel:`Invoicing Policy`: set as :guilabel:`Prepaid/Fixed Price`;
+- :guilabel:`Create on Order`: select :guilabel:`Nothing`;
+- :guilabel:`Sales Price`: type in the **amount** of the top-up.
+
+.. note::
+   If you wish to have eWallet **top-ups** of different amounts, repeat the process and modify the
+   :guilabel:`Sales Price` accordingly.
+
+Once the **top-up** is created, go to :menuselection:`Website app --> eCommerce --> Gift Cards &
+eWallet` to :guilabel:`Create` an eWallet **program**. The following configuration options are
+available:
+
+- :guilabel:`Program Name`: enter the name you want;
+- :guilabel:`Program Type`: select :guilabel:`eWallet`;
+- :guilabel:`eWallet Products`: select the eWallet **top-up(s)** created earlier;
```

Comment:
I don't see this field (see screenshot of what I see below)

Reply:
Marking this as 'resolved' since it is the one I showed you on your screen directly